pkg/daemon/daemon.go

```diff
@@ -588,26 +586,33 @@ func (p *ptpProcess) cmdRun(stdoutToSocket bool) {
 					if regexErr != nil || !logFilterRegex.MatchString(output) {
 						fmt.Printf("%s\n", output)
 					}
-					source, ts2phcsOffset, state, iface := extractMetrics(p.messageTag, p.name, p.ifaces, output)
-					var ts2phcState event.PTPState
-					if state == FREERUN {
+					source, ts2phcsOffset, _, iface := extractMetrics(p.messageTag, p.name, p.ifaces, output)
+					if iface != "" {
+						var ts2phcState event.PTPState
 						ts2phcState = event.PTP_FREERUN
-					} else if state == LOCKED {
-						ts2phcState = event.PTP_LOCKED
-					}
-					if source == ts2phcProcessName {
-						p.eventCh <- event.EventChannel{
-							ProcessName: event.TS2PHC,
-							State:       ts2phcState,
-							CfgName:     p.configName,
-							IFace:       iface,
-							Values: map[event.ValueType]int64{
-								event.OFFSET: int64(ts2phcsOffset),
-							},
-							ClockType:  clockType,
-							Time:       time.Now().Unix(),
-							WriteToLog: true,
-							Reset:      false,
+						updateClockStateMetrics(p.name, iface, FREERUN)
```

Comment:
this line should move to line 599 in an `else` statement of checking Max/MinOffsetThreshold

Reply:
since we are bringing in max/min threshold values,They should be applied for all configurations